plugin.rb

```diff
@@ -78,13 +78,6 @@
     end
   end
 
-  on(:category_created) do |category|
-    about_post = category.topic.posts.first
-    revisor = PostRevisor.new(about_post, about_post.topic)
-    about = "#{category.custom_fields["introduction_raw"]}\n\n#{category.custom_fields["tenets_raw"]}\n\n#{about_post.raw}"
```

Comment:
Are you sure that we need to remove `introduction_raw` from here? It seems like this is the only place where we using it. Maybe we need to write something like this?:

```
  on(:category_created) do |category|	
    about_post = category.topic.posts.first	
    revisor = PostRevisor.new(about_post, about_post.topic)
    about = category.custom_fields["introduction_raw"].presence || about_post.raw
    revisor.revise!(about_post.user, { raw: about }, skip_validations: true)	
  end
```